src/Seq.Forwarder/Config/SeqForwarderConfig.cs

```diff
@@ -45,8 +48,28 @@ public static SeqForwarderConfig ReadOrInit(string filename)
             }
 
             var content = File.ReadAllText(filename);
-            return JsonConvert.DeserializeObject<SeqForwarderConfig>(content, SerializerSettings) ??
-                throw new ArgumentException("Configuration content is null.");
+            var seqForwarderConfig = JsonConvert.DeserializeObject<SeqForwarderConfig>(content, SerializerSettings);
```

Comment:
I think throwing when `DeserializeObject` returns null might be safer here, since it'd indicate either a bug or malformed configuration file, both of which we might want to know about. The later check won't be triggered if any environment variable is set.

Reply:
That makes sense. I thought the config file might be optional but on second thoughts just the env vars are. I've made these two changes now (https://github.com/datalust/seq-forwarder/pull/57/commits/1be87596361c927bfef9c3f74423cf967fd95c2c)